### DISCUSSION TITLE
Is Pybullet compatible with latest Tensorflow version? / Configurations

### DISCUSSION THREAD

**Discussion by rohit-kumar-j** [bulletphysics/bullet3#3381] [General]
Hi! 
I found  a question in stack overflow: 
https://stackoverflow.com/questions/64216867/error-while-running-deepmimic-example-in-pybullet-with-tensorflow
I was wondering if there has been an update that might be able to let us run TensorFlow versions that are above 1.13(Is it required to use version >=1.13)?

Can we simply do a `python3 -m pip install tensorflow-gpu` ?

I am running Pybullet on Ubuntu 20.04 LTS and need to do a tedious configuration corresponding to the ones [specified](https://www.tensorflow.org/install/source#linux) below in order to use TensorFlow with an RTX 2070:   

![image](https://user-images.githubusercontent.com/37873142/107360869-87f1b680-6afc-11eb-94fb-b6e674b392c8.png)

Similar Issue: #3123 
Warm Regards,
Rohit

**Comment by erwincoumans** [comment]
No, it is not compatible, you need to use an older TensorFlow version. Do you offer to upgrade all use of Tensorflow in Bullet to be compatible with the latest, and become maintainer?

**Reply by rohit-kumar-j** [reply]
@erwincoumans I am planning to re-create deep mimic. Hence, the question. I suppose if I use tf-gpu to create deep mimic, then I could, in theory, work out the rest of the examples. Will create pull requests when I do.⚡Perhaps, become a maintainer then.
